packages/compiler/test/path-utils.test.ts

```diff
@@ -165,13 +165,6 @@ describe("compiler: path utils", () => {
     strictEqual(getBaseFileName("http://server/"), "");
     strictEqual(getBaseFileName("http://server/a"), "a");
     strictEqual(getBaseFileName("http://server/a/"), "a");
-    strictEqual(getBaseFileName("/path/a.ext", ".ext", /*ignoreCase*/ false), "a");
```

Comment:
was there anything that changed here or just removed the additional functionality that we didn't use?

Reply:
I made getAnyExtensionFromPath return extension lowercased and it was confusing that there was this different ignoreCase concept that we weren't using so I deleted that.

Reply:
ah didn't noticed this was getting called from the other function